### PR TITLE
Add terminal directive explanation

### PR DESCRIPTION
**Pull request by kazupon**
See:
https://github.com/vuejs/vue/pull/2272

### CODE REVIEW

**Comment by ludo237** [comment]
:+1:

File: src/guide/custom-directive.md [hunk 1]
```diff
@@ -277,8 +277,50 @@ Vue.directive('my-directive', {
 
 Use this wisely though, because in general you want to avoid side-effects in your templates.
 
+### terminal
+
+In some cases, we may want to used as the **terminal** directives. For example, you may want to perform custom directive than normal directive like `v-if` or `v-for`. If you want to do so, you need to pass in `terminal: true` in your directive definition.
```
Comment (fullfs) [inline src/guide/custom-directive.md]: "custom directive than normal directive" -> "custom directive instead of normal directive" 
sounds better to my ear

Comment (fullfs) [inline src/guide/custom-directive.md]: Can't get it ) What is terminal directive? How is it different from non-terminal one? Why would I want to make a terminal? 
Tbh I don't know myself what it is and can't understand it from the text )

Comment (kazupon) [inline src/guide/custom-directive.md]: Thank you for your review!!
Sorry, poor documentation and english. :bow: 
I'll try to fix.

**Comment by yyx990803** [comment]
Thanks @kazupon ! I'm releasing 1.0.19, so I'll merge this and edit it before release.

File: src/guide/custom-directive.md (hunk 1 above)
Comment (fullfs) [inline src/guide/custom-directive.md]: @kazupon Oh don't worry ) I'm not so very good in english as well. It's not my native :)

**Pull request merged**